bitnami/valkey/templates/sentinel/hpa.yaml

```diff
@@ -11,6 +11,7 @@ metadata:
   namespace: {{ include "common.names.namespace" . | quote }}
   labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
     app.kubernetes.io/component: replica
+    app.kubernetes.io/part-of: valkey
```

Comment:
For consistency with other charts. Once there is a vpa + hpa, then hpa goes under autoscaling.hpa